docs/extending.md

```diff
@@ -13,69 +9,63 @@ Useful resources are:
 - The [nf-test documentation](https://code.askimed.com/nf-test/docs/getting-started/)
 - The [nf-core slack](https://nf-co.re/join), particularly the [multiplesequencealign channel](https://nfcore.slack.com/archives/C05LZ7EAYGK). Feel free to reach out!
 
-The pipeline consists of four different subworkflows:
+## Adding an aligner
 
-1. Compute the guide trees for guide tree-based methods.
-2. Perform the MSAs.
-3. Evaluate the produced MSAs.
-4. Compute statistics about the input dataset.
+This steps will guide you to include a new MSA tool into the pipeline. Once done, this will allow to systematically deploy and benchmark your tool against all others included in the pipeline.
```

Comment:
Maybe add here also that you can contribute back to the pipeline
```suggestion
These steps will guide you to include a new MSA tool into the pipeline. Once done, this will allow you to systematically deploy and benchmark your tool against all others included in the pipeline. You are also welcome to contribute back to the pipeline if you wish.
```